### PR TITLE
Quarkchain

### PR DESCRIPTION
**Pull request by coinmandeer**
QKC

### CODE REVIEW

File: db/projects/monetha/asset-mth-logo-square.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg height="32" viewBox="0 0 32 32" width="32" xmlns="http://www.w3.org/2000/svg"><g fill="none"><circle cx="16" cy="16" fill="#104fca" r="16"/><g fill="#fff" transform="translate(7 9)"><path d="m0 .56757699 4.2345 7.10328525v7.32913776c-2.33865 0-4.2345-1.8360837-4.2345-4.101075z" fill-opacity=".5"/><path d="m13.76487 7.66320318 4.2319086-7.09893822h.0025914v10.33466004c0 2.2649913-1.89585 4.101075-4.2345 4.101075z" fill-opacity=".6"/><path d="m8.99739174 7.45835186-2.11725174 3.55161324c-.43914-.1106914-.91125-.5363451-1.41633-1.27696103l-5.46381-9.16542708c2.025-1.13245206 4.61511-.46037188 5.78439 1.50113306z" fill-opacity=".8"/><path d="m12.21561 2.05990703c1.16928-1.96150494 3.75939-2.63358512 5.78439-1.5010459l-5.46381 9.16542707c-.75762 1.1109239-2.0583 1.843928-3.53583 1.843928-.65968947.0009185-1.3103983-.1480775-1.89981-.4350088l-.22041-.1232423z"/></g></g></svg>
```
Comment (cryptomari0) [inline db/projects/monetha/asset-mth-logo-square.svg]: Maybe the full logo is more like this one? asset-mth-logo-full.svg
https://www.monetha.io/3a89703038409d26fd9db1370501dfba.svg

The one you are mentioning here is based on my opinion asset-mth-logo-alternative.svg

File: db/projects/monetha/asset-mth-logo-alternative.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...249.5C0 1509.3 210.6 1720 470.5 1720V63.7H0v1185.8z"/><linearGradient id="SVGID_2_" gradientUnits="userSpaceOnUse" x1="1764.65" y1="-888.302" x2="1764.65" y2="767.998" gradientTransform="translate(0 952)"><stop offset="0" stop-color="#16ed8c"/><stop offset="1" stop-color="#0767fc"/></linearGradient><path class="st1" d="M1999.9 1249.5c0 259.9-210.7 470.5-470.5 470.5V63.7h470.5v1185.8z"/><linearGradient id="SVGID_3_" gradientUnits="userSpaceOnUse" y1="-287.849" x2="1235.6" y2="-287.849" gradientTransform="translate(0 952)"><stop offset="0" stop-color="#0b46aa"/><stop offset="1" stop-color="#1455e1"/></linearGradient><path class="st2" d="M642.7 236.4C512.8 11.3 225-65.8 0 64.1l607.1 1051.6c84.2 127.5 228.7 211.6 392.9 211.6 73.3.1 145.6-17 211.1-49.9l24.5-14.1L642.7 236.4z"/><linearGradient id="SVGID_4_" gradientUnits="userSpaceOnUse" x1="764.4" y1="-288.851" x2="2000" y2="-288.851" gradientTransform="translate(0 952)"><stop offset="0" stop-color="#0b46aa"/><stop offset="1" stop-color="#1455e1"/></linearGradient><path class="st3" d="M1357.3 235.3C1487.2 10.3 1775-66.8 2000 63.1l-607.1 1051.6c-84.2 127.5-228.7 211.6-392.9 211.6-73.3.1-145.6-17-211.1-49.9l-24.5-14.1 592.9-1027z"/></svg>
```
Comment (cryptomari0) [inline db/projects/monetha/asset-mth-logo-alternative.svg]: This one seems for me to be asset-mth-logo-square.svg

File: db/projects/pundix/project.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
+name: Pundi X
+web:
+  - 'https://pundix.com/'
+start_date: '2016-01-01'
+description: 'Pundi X aims to bring in the next billion crypto users as it allows users to buy, sell, use cryptocurrency anywhere and anytime.'
```
Comment (cryptomari0) [inline db/projects/pundix/project.yaml]: Maybe I agree with Tree in this opinion. _aims to bring in the next billion crypto users_ This is nothing like description, but more like their advertise. But anyway, I know how hard is to write some normal description. So maybe better not to fill this field?

File: db/projects/quarkchain/project.yaml [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+      - title: 'Public participants'
+        percent: 20
+      - title: 'Mining & community'
+        percent: 45
+      - title: 'Team'
+        percent: 15
+      - title: 'Advisors'
+        percent: 5
+      - title: 'Fund'
+        percent: 15
```
Comment (cryptomari0) [inline db/projects/quarkchain/project.yaml]: It's nothing like mistake, but just for your information. For this field you can choose either percent: number (represent the percent of the token as you used) or amount: number (represent the total number of the tokens).

File: db/projects/monetha/asset-mth-logo-square.svg (hunk 1 above)
Comment (coinmandeer) [reply db/projects/monetha/asset-mth-logo-square.svg]: switched alternative to square

File: db/projects/pundix/project.yaml (hunk 3 above)
Comment (coinmandeer) [reply db/projects/pundix/project.yaml]: I changed description to "Zilliqa is project focused on scalability and sharding research"

File: db/projects/quarkchain/project.yaml (hunk 4 above)
Comment (coinmandeer) [reply db/projects/quarkchain/project.yaml]: ok, I found percentages easier to dig up

**Pull request merged**